libs/nillion-chain/client/src/client.rs

```diff
@@ -438,3 +440,33 @@ impl NillionChainClient {
         client.abci_query(path, data, None, false).await.map_err(Into::into)
     }
 }
+
+#[cfg(test)]
+mod tests {
+    use rstest::rstest;
+    use crate::client::NillionChainClient;
+
+    #[rstest]
+    #[case(4000, 0.025, 0, 100, "No rounding up for exact multiple")]
+    #[case(4000, 0.025, 1, 101, "No rounding up for exact multiple")]
```

Comment:
You moved the strings but they're still outdated, e.g. this says "no rounding up" but there _is_ round up.